### DISCUSSION TITLE
Ban from the discord???

### DISCUSSION THREAD

**Discussion by XenoXzOFF** [pterodactyl/panel#5116] [Support] [answered]
Hello, I was banned from your discord but I don't know why.
I just asked a question then left the discord because I had too many.
I think you have the wrong person.
Can you unban me please.
THANKS.

Discord: XenoXzOFF

**Comment by rowansc1** [comment, marked as answer]
Hi @XenoXzOFF, you were removed from the Discord by one of the moderators for attempting to get support for install scripts after being told that they are not supported here, and that you need to go to the script developers discord for support.
